packages/cloud-config/definition.yaml

```diff
@@ -1,3 +1,3 @@
 name: cloud-config
 category: system
-version: 0.7+1
+version: "0.8"
```

Comment:
why the quotes?

Reply:
That's a yaml annoying thing... the version is a string because it is semver compliant ( so can contain things like `+`, `-` and string characters) but gets parsed as float if you don't quote it explictly in this case (the `+` removes the ambiguity between string and numbers, but `0.8` it looks like a float )